playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/pre.yml

```diff
@@ -110,8 +110,8 @@
   # are modified to use the correct image tag.  However, this can trigger a
   # docker restart if new configuration is laid down which would immediately
   # pull the latest image and defeat the purpose of these tasks.
-  - openshift_cli
-  pre_tasks:
+  - openshift_docker
```

Comment:
@dgoodwin What about this...

- oo_first_master:
  - Pre-upgrade checking prior to updating the docker config
  - apply openshift-docker role
  - Query new versions
- oo_all_hosts - oo_first_master
  - Pre-upgrade checking prior to updating the docker config
  - apply openshift_docker role
  - Validate new versions from oo_first_master are available
...

That could limit the potential issues that could arise from attempting to modify the docker configs all around.